substrate/pkg/controller/substrate/cluster/addons/awsebscsi.go

```diff
@@ -0,0 +1,32 @@
+package addons
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/awslabs/kit/substrate/pkg/apis/v1alpha1"
+	"github.com/awslabs/kit/substrate/pkg/utils/helm"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type AWSEBSCSIDriver struct{}
+
+func (l *AWSEBSCSIDriver) Create(ctx context.Context, substrate *v1alpha1.Substrate) (reconcile.Result, error) {
+	if !substrate.Status.IsReady() {
+		return reconcile.Result{Requeue: true}, nil
+	}
+	if err := helm.Apply(ctx, &helm.Chart{
+		Namespace:  "kube-system",
+		Name:       "aws-ebs-csi-driver",
+		Repository: "https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/download/helm-chart-aws-ebs-csi-driver-2.6.3",
+		Version:    "2.6.3",
+		KubeConfig: *substrate.Status.Cluster.KubeConfig,
+	}); err != nil {
+		return reconcile.Result{}, fmt.Errorf("applying chart, %w", err)
```

Comment:
We should specify chart name in the error return because if a generic error is returned from `Apply` it will be hard to tell which chart failed.

Reply:
It's covered by the current logging by saying that the AWSEBSCSIDriver reconciler failed.